### DISCUSSION TITLE
Robust Server for multi sessions!

### DISCUSSION THREAD

**Discussion by Saifallak** [WhiskeySockets/Baileys#28] [General Questions]
Looking for a Robust Server (nodejs or whatever) that can manage sessions (create sessions and destroy them) and offers api to use with other systems,

typically looking for something like [@wppconnect-team/wppconnect-server](https://github.com/wppconnect-team/wppconnect-server) but for Baileys,

reason is that i want to use Mobile Native API which will be made by #2,

so looking for something to start with and I can improve more by the time,

so if someone is using some kind of server that is stable would be appreciated 

i dont want to make a bot of it, just want to send some messages when some events happens in the system.

**Comment by erickythierry** [comment]
I made a copy of this project a few months ago and created my own server using it as a base.
currently they are archived because the main repository of baileys has deleted, so the creator has given up, but with some adjustments it is still possible to use.

[baileys-api](https://github.com/ookamiiixd/baileys-api)

**Reply by Saifallak** [reply]
thanks, will try it out

**Comment by w3nder** [comment]
[code-chat-br](https://github.com/code-chat-br/whatsapp-api)

I believe you will like it :)

**Reply by Saifallak** [reply]
Looks great, will try it for sure,
but if you're using it, what is 

```env
# Determine how long the instance should be deleted from memory in case of no connection.
# Default time: 5 minutes
# If you don't even want an expiration, enter the value false
DEL_INSTANCE: 5 # or false
```

will it delete it and i have to re-scan or what?
